### DISCUSSION TITLE
체스 미션을 위한 웹 기본 강의 | 2022.03.22

### DISCUSSION THREAD

**Discussion by wmakerjun** [woowacourse/retrospective#11] [General]
# 🖥️ 강의 자료
[슬라이드](https://docs.google.com/presentation/d/1iiHVRy5Fx_goQ0actrE216FeArDNmNBntHHdTJOJVIE/edit?usp=sharing)

<br>

# 🎯 우리의 목표
복잡하고 어려운 체스 게임을 **동작 가능한 가장 작은 버전**으로 만들어 보며 웹의 동작을 이해한다. 

<img width="300" alt="스크린샷 2022-03-18 오전 8 42 56" src="https://user-images.githubusercontent.com/50367798/159388171-b318cbbd-7599-4b91-a38e-93a0d3e7f358.png">

<br>

# 🚀 미션
### 1단계 - 필수 
`form`을 구현하여 서버에 chess 말의 이동 명령을 전송하고, 데이터를 응답 받아서 갱신한다. 이 때 [handlebars](https://github.com/jknack/handlebars.java) 문법을 통해 ui는 자동으로 rendering 되므로, `form`에만 집중한다.

### 2단계 - 선택
클릭 이벤트와 같은 사용자 이벤트를 다뤄서 체스를 구현해본다. 그리고 json 형태로 데이터를 서버와 주고 받으며 비동기의 개념을 이용하여 화면을 렌더링한다.
  - 사용자의 이벤트를 받아, event 객체에서 얻을 수 있는 데이터들을 다룬다.
  - [fetch](https://developer.mozilla.org/ko/docs/Web/API/Fetch_API) API를 이용해서 데이터를 주고 받는 과정에서 필요한 개념들을 이해하고 활용한다.
 
<br>

## 🤚🏻 미션 진행 방법
- [web-chess-basic](https://github.com/woowacourse/web-chess-basic)를 fork하여 구현한다.
- 페어와 함께 진행한다. `codeWithMe` 또는 화면 공유로 한 사람의 컴퓨터에서 같이 작업한다.
- 진행이 빠르게 끝났다면 2단계 미션을 도전한다. (2단계는 프론트엔드 작성에 익숙한 크루들이 아니라면 지금은 신경쓰지 않아도 괜찮습니다.)

### 미션 진행시 체스판의 position은 아래의 규칙을 따른다.
<img width="300" alt="스크린샷 2022-03-18 오전 8 42 56" src="https://user-images.githubusercontent.com/50367798/159388274-a79522f8-ebc5-417d-8538-40f1464fd4a7.png">

<br>

# 📝 회고
- 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.

**Comment by yhh1056** [comment]
## 썬 & 호호

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- move.http 를 참고할 생각을 못 했다.
- 자바 코드를 분석해야했다.
- id와 name의 차이를 몰라 전송을 하는데 시간이 걸렸다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 체스미션 큰일났다....
- html 지식이 이렇게 없는 줄 몰랐다. (호호)
- 네오의 힌트가 있어서 빠르게 해결할 수 있었다. 네오 없었으면 점심 못 먹었다.
- 2단계는 아직 무리...😅

**Comment by YJGwon** [comment]
## 돌범 & 이스트 & 포키
#### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 버튼까지 다 만들었는데 안돼서 헤맸다. action까지는 찾았는데, command 찾는게 오래 걸렸다.
- fetch랑 merge 아직도 헷갈린다. 왜 하나는 띄워쓰고 하나는 슬래시인가...
#### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 이스트: 3명이라 불편할 줄 알았는데 집단지성의 힘으로 아주 잘 해냈다!
- 돌범: 인사를 나중에 하고 미션부터 수행했는데, 이미 페어 프로그래밍 경험이 쌓여서 불편함이나 어색함 없이 진행되었다.
- 포키: HTML을 분명 해봤는데.... 역시 경험과 지식은 따로구나를 느꼈다.

**Comment by sinb57** [comment]
## 이프 & 오리

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 태그를 하나도 모른다.
- input의 id값이 command여야 하는 것을 뒤늦게 알았다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 이프의 버스를 받아야겠다. (오리)
- HTML에 대해 친숙하게 다가갈 수 있었다. (오리)
- 오리의 버스를 받아야겠다. (이프)
- 웹 구조를 파악하는 것은 역시 재밌다. (이프)

**Comment by summerlunaa** [comment]
## 파랑 & 매트

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- HTML을 잘 몰라서 찾아보면서 한다고 쉽지 않았다.
- input 태그에 name을 빼먹어서 자꾸 오류가 났다.
- 2단계도 한 번 해보려고 했는데 어려워서 실패했다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 파랑: 프론트 예전에 배울 때 좀 열심히 들을 걸 후회했다.
- 매트: 오랜만에 프론트를 해서 쉽지 않았고 체스 미션이 걱정된다.

**Comment by sc0116** [comment]
# 짱구 & 찬

## 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- `action`을 써서 url은 잘 가는데 동작이 안되서 당황했다.
- `Request`에 `command`를 찾지 못해서 좀 헤맸다....

## 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 우엑 HTML 싫다...
- 찬 캐리로 `command`를 찾아서 다행이다
- 프론트엔드 어렵지만 최종 결과에선 마우스로 이동시키는 것을 구현해보고 싶다

**Comment by Juhyung990122** [comment]
## 릭 & 열음

### 오늘 강의와 미션에서 과정을 진행하면서 어디서 어려운 점을 느꼈나요?
- 속성명을 찾기가 어려웠다.
- action 지정하는게 낯설었다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 릭 : 백엔드 하길 잘했다..^^
- 열음 : 체스... 큰일났다... 너무 즐거워서... 큰일났다...

**Comment by kbsat** [comment]
### 레넌, 로마

**오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?**
- 오랜만에 해서 그런지 다소 낯설게 느껴졌다. 어떤 곳으로 요청을 보내야하는지 찾기 어려웠다.

**오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.**
- 웹의 근본에 대해서 알게될 수 있어서 즐거웠다. 간만에 HTML 하니 재밌다.

**Comment by Seongwon97** [comment]
# 차리 & 렉스

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 친숙하지 않은 HTML을 작성하는데 어려움을 느꼈습니다.
- HTML을 만들기는 했는데 Server로 넘어가는 과정을 확실히 이해하지 못하여 디버깅이 어려웠습니다.
- 2단계는 포기...🤯

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 차리 : 내가 프론트를 안한 이유가 있었네 😠
- 렉스 : 그동안 HTML을 공부하지 않은 나 자신을 원망하였습니다..앞으로 공부해보겠습니다.

**Comment by 2yujeong** [comment]
## 범고래 & 이브
### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- Html 언어가 익숙하지 않아서 간단했지만 헷갈렸다. 그래도 재밌었다.
- Spark같은 생소한 라이브러리를 이용한 코드 때문에 헷갈렸다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 제대로 습득하지 않은 지식은 활용하기 어렵다.
- 백엔드와 달리 눈에 보여서 재밌었다. 
- 프론트, 백엔드 둘 다 재밌었다.

**Comment by yangdongjue5510** [comment]
# 에덴 & 클레이

## 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?

- HTML이 낯설었습니다.
- 만들어진 어플리케이션의 흐름을 파악하기 어려웠습니다.
- 2단계는 무슨 말인지 이해가 안됐다.
- 새로고침을 안해서 한참을 헤맸다.

## 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.

- 클레이 : 새로고침을 자주 하자!
- 에덴 : 체스 어떻게 하지....

**Comment by seong-wooo** [comment]
# 아서 & 배카라

## 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?

아서 : 어떻게 작동하는 지 몰라서 어디를 수정해야할 지 잘 몰랐다.
배카라 : html을 아직 잘 모르기때문에 어려웠습니다. ㅜ 

## 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.

아서 : 체스를 웹으로 구현할 수 있을까... ?
배카라 : 아서를 믿습니다. 화이팅~

**Comment by tco0427** [comment]
### 디우 & 잉

---

**오늘 강의와 미션에서 과정을 진행하면서 어디서 어려운 점을 느꼈나요?**
    
- 자바 코드를 분석하는데서 어려움이 있었다.
(구체적으로는 input 태그의 name 속성이 무엇인지를 찾는게 어려웠다.)
    
- 태그 문법(?)이 잘 기억이 안나서 혼용하였다.
    
- HTML이 처음이라서 기본적인건데도 이해하는데 좀 걸렸다.
    
**오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.**
    
- HTML을 공부했었지만 자주 사용하지 않다보니 어색한 느낌이 들었다.
    
- 아주 기초적인 거지만 새로운 지식을 얻을 수 있어서 좋았다.
    
- 체스 미션 어떻게 하지..?

**Comment by Seungpang** [comment]
## 베루스 & 승팡
### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
+ 스파크의 Request의 body가 지연로딩된다는 사실을 모르고 있어서 디버깅할때 헤맨 부분이 어려웠습니다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
+ 베루스: 체스 이거 어쩌지...
+ 승팡: 이걸 웹으로 구현한다고....?

**Comment by kth990303** [comment]
## 슬로 & 케이
### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 슬로: 프론트 태그들이 낯설었다. 그리고 button을 만들고 action 사용하는 것까지 했는데 이름을 command로 하는 것을 몰라서 방법을 찾는데 오래걸렸다.
- 케이: 자바 코드를 확인하여 move로 보내야되는점, name을 설정해주지 않으면 전달이 안되는 점에서 어려움을 겪었다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 슬로: 프론트 공부를 조금이라도 해놓을걸... 케이랑 페어가 돼서 너무 기분이 좋다!
- 케이: 체스 웹으로 구현을 잘할 수 있을까? 그래도 한번 해두면 웹 지식 많이 늘듯? 슬로랑 스터디도 같이 발표하고 체스미션도 페어가 돼서 되게 신기하고 좋았다. 브멘...

**Reply by YJGwon** [reply]
브멘...

**Comment by dongho108** [comment]
## 🐢아키 & :fire:애쉬
## 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 아키 : 폼 태그로 데이터를 잘 보냈다고 생각했는데 콘솔에 찍어보니 null로 표시되었습니다.  
나중에 알고보니 input 태그에 name으로 키값을 입력해야 된다는 사실을 알았습니다.
- 애쉬 : 분명 post /move 까지 잘 가는데 body에 데이터가 담기지 않았습니다.
결국엔 move.http를 참고해서 command로 인식할 수 있도록 name을 지정해주어야 한다는 것을 알았습니다.

## 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 아키 : 에러와 마주했을 때 해결하기위해서는 동료와 생각을 공유하는 과정이 중요하단 것을 새삼 느꼈습니다.
- 애쉬 : 생각보다 쉽게 끝날 줄 알았는데 오래걸려서 아쉬웠습니다. 
그리고 꼭 spring, springboot가 아니더라도 자바로 웹 어플리케이션을 실행할 수 있다는 것이 신기했습니다!

**Comment by yh20studio** [comment]
## 라쿤 & 티키

###  오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 이미 작성된 코드를 따라가면서 이해하는 것이 어려웠다.
- HTML 코드가 익숙하지 않았다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 체스는 쉽지 않을 것 같다.
- 페어와 즐거운 3주를 보내며 영혼의 듀오가 될 것 같다.

**Comment by injoon2019** [comment]
# 후디 & 토르
## 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
  - 후디: 스파크 동작을 모르다보니 파악하는데 어려움이 있었습니다.
  - 토르: 저도 서버쪽 코드도 이해하기 어려웠고, HTML도 잘 사용해보지 않아 낯설었습니다.
   
## 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 후디: 같은 조 사람이랑 페어가 돼서 좋았습니다. 체스 미션 어려울 것 같아요!
- 토르: 저도 후디랑 페어가 돼서 기분이 좋습니다. 실제 웹에서 사용되는걸 보니 조금 더 신기하고 관심이 가네요.

**Comment by gudonghee2000** [comment]
## 레오 & 리버
---
### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 레오 : HTML이 익숙하지 않아서 tag를 찾아보는데 시간이 많이 걸렸다. 
- 리버 : HTML을 잘 몰라서 form과 input의 기능을 찾기가 어려웠다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 레오 : 체스미션 쉽지 않을것 같다..!
- 리버 : 벌써 체스미션이 무섭다..!

**Comment by tonic523** [comment]
## 토닉 & 크리스

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
웹 동작 방식의 기본을 아는게 어려웠다.
html이랑 java application이 어떻게 데이터를 전달하는지 이해하는 부분이 어려웠다.
form의 태그 중 enctype을 몰라서 해멨다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
체스 큰일났다...

**Comment by OzRagwort** [comment]
# 어썸오와 알린

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
코드의 실행 과정을 이해하고 나니 쉽게할 수 있었다.
처음에 input에 name=”command”를 쓰지 않았는데 name이 없으니까 제대로 동작하지 않았다.
예시에서 command=a1,a2로 되어있는게 무슨 의미인지 알고나니 쉬웠다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
HTML 공부를 조금 해야겠다.
2단계는 Javascript를 이용해야 하는 것 같은데 잘 몰라서 감도 안왔다.

**Comment by RIANAEH** [comment]
### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 주디 : HTML에 익숙하지 않아서 모든게 새로운 것이 어려웠습니다.
- 엘리 : 적절한 속성 값을 지정해주는게 어려웠습니다. Not Found..

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 주디 : 웹공부를 더 열심히해야겠다고 느꼈고, 험난한 미션이 될 것 같습니다.
- 엘리 : 역시 html 잘 안맞는다.. 백엔드 좋아.. 

체스 미션 화이팅♟

**Comment by jojogreen91** [comment]
# 후니 & 조조그린

### 오늘 강의와 미션에서 과정을 진행하면서 어디서 어려운 점을 느꼈나요?

- input 태그의 Name 특성에 "command" 를 입력해주지 않아서 body 로 `null` 값이 들어와서 에러가 발생했다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.

- 후니 : html 공부를 하자!
- 조조그린 : 반갑다 html 아!

**Comment by ulimy** [comment]
## 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
input name에 command를 적어야 한다는 사실을 알지 못해 난항을 겪었습니다..
enctype... text/plain... 실수로 삭제했다가 난항을 겪었습니다..

## 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
찬의 도움 덕에 해결 할 수 있었습니다. 백엔드 하길 잘했다......... 백엔드.. 사랑해요.......

**Comment by woong7** [comment]
## 라라 & 판다

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 라라: 프론트 개발을 처음 하다보니 어디부터 손대야 할지도 당황스럽고 어려움을 느꼈다....ㅎㅎ
- 판다: html에 대한 기본적인 지식의 부재를 느꼈습니다. 내가 지금 뭘 모르는 건지 모르겠어요
 
### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 라라: 프론트는 어렵다... 백엔드가 훨씬 재밌구나 ^_^ 를 느꼈다.
- 판다: 역시 체스미션 쉽지 않을듯... 빡세게 해야지

**Comment by ldk980130** [comment]
### 알파와 더즈

**오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?**
- 알파 : 자바와 html이 공통적으로 정한 약속을 처음에 찾기 어려웠다.
- 더즈 : 아무런 정보 없이 시작하여 "commend" 키워드를 준의 도움 없이 찾지 못한 게 아쉬웠다.

**오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.**
- 알파 : 앞으로 고생 길이 훤하다
- 더즈 : 체스와 웹 정말 머리 아플 것 같다ㅎㅎ

**Comment by wishoon** [comment]
<h2> 헌치 & 루키 </h2>

<h3>오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?</h3>

헌치 : name 속성과 action 속성의 작동방식을 잘 몰라서 어려웠다.
루키 : input 태그의 name 속성이 기억나지 않아 미션을 해결하는데 어려움을 겪었었다.

<h3>오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.</h3>

헌치 : html에서 데이터 보내는 형식에 대해 알게되어 좋았고 코드를 작성하며 작동하는 모습을 바로 볼수있어서 좋았다.
루키 : 체스 미션 큰일났을수도..

**Comment by her0807** [comment]
## 연로그 & 수달 🤓 
### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 연로그 : html을 써본적은 있으니까 만만히 봤는데 큰코 다쳤다....
- 수달 : name 태그가 아니라, id 태그를 사용해서 헤맸다 .. 

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 연로그 : 체스를 싫어하게 될 것 같다. :
- 수달 : Html 을 다시 공부해야겠다고 생각했다.

**Comment by Youngyoon-1** [comment]
## 칙촉 & 유콩

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 칙촉: html 태그에 대한 지식 부족
- 유콩: 고작 몇달 전에 썼던 js 가 기억나지 않아 아쉬웠다.
### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 칙촉: 체스. 큰일이다..
- 유콩: 오랜만에 콘솔창이 아닌 “화려한” 웹 화면을 볼 수 있어 좋았고 체스 미션이 기대된다!^^

**Comment by chawani** [comment]
### 쿤 & 마루

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
* **쿤** : HTML 태그에 대한 개념이 부족해서 태그에 대한 이해를 하는데 시간이 오래 걸린 것같다. input 태그의 name 속성을 추가하지 않아 오류가 발생했는데, 더 공부해봐야 할 것 같다!!
* **마루** : 태그를 예전에 너무 잠깐 써봐서 다시 개념을 상기하는 데 어려움이 있었고, name에 command를 입력해줘야 하는지 몰라서 헤맸다..
### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
* **쿤** : 체스 미션이 무섭지만 해쳐나가봅시다!!
* **마루** : html이 쉬워보이면서도 어렵다. 많이 공부해야할 것 같다. 쿤 만나서 반갑고, 체스 화이팅 해봐요😎

**Comment by Gyuchool** [comment]
## 봄 과 기론
오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?

봄 : input에서 name을 잘 적어야했다.
기론 : req, requset, board등 다했는데 command를 못 찾아서 아쉬웠다.

오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.

봄 : 체스 미션이 많이 걱정된다.
기론 : 앞으로가 느낌이 싸하다.

**Comment by HJ-Rich** [comment]
https://github.com/HJ-Rich/web-chess-basic/tree/main

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?

루나
- HTML 만만하게 봤다가 한 대 맞은 듯... 리차드 버스 달달..

리차드
- 버스 요금 내세요


### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.

루나
- 프론트 크루들이 좀 더 이해하기 쉬운 코드를 만들어야겠다...

리차드
- 프론트엔드 리스펙!

**Comment by bugoverdose** [comment]
## 앤지와 정

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- 생각보다 우리가 form 관련 작업을 많이 안해봤다는 점을 깨달았다. 항상 구현한 코드를 그대로 복붙하다보니 경험이 제대로 쌓이지 않은 것 같다.
- 뿌리가 좀... 얕을지도?
- 체스미션 하다보면 뿌리가 깊어질지도?

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 경험해보지 못한 환경이라 당황하게 된 것 같다.
- 은근 재밌었다. 앞으로도 더 열심히 해야겠다. 
- 불타오른다. 빠이어🔥

**Comment by progress0407** [comment]
### 소주캉과 필즈

---

#### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?

- 올바른 변수명인 `command`를 계속찾지 못했다..ㅎㅎ  (스파크 웹 프레임워크가.. 먼지 몰라서..)
- 태그 자체에 익숙하지 않아 곤경을 느꼈다 ㅠㅠ.. 오류가 나도 오작동을 해도 어디서부터 디버깅할 지 어려움을 느꼈다

#### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 오늘 같이 하면서 즐거웠다.. ㅎㅎ
- 같이 ~~음코~~하면 더욱 **재미**있을 것 같다 :)
- 팀뷰어를 키면 음성이 낮게 들려서 불편했다... 아쉬웠다 ㅠㅠ..
  - 코드위드미
- 현재 어느 부분에서 막히고 있는지 고민을 좀 더 잘 전달할 필요가 있어 보인다 !

**Comment by hamcheeseburger** [comment]
## 써머와 코린

**오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?**
- 서버 코드가 처음보는 형태라 분석하기 어려웠다!
- 서버 전송하는게 너무 어려웠다 ㅜㅜ (fetch라는 기본 api가 있는 줄 몰랐다)


**오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.**
- 체스 미션 큰일 났다..
- js 너무 오랜만에 해서 반가운데 짜증났다!
- 오랜만에 낯선 것을 해서 환기가 됐다!
- 네오에게 감사..🙇🏻‍♀️

**Comment by meatsby** [comment]
## 알렉스 & 쿼리치

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?

- 당연히 json 포멧인 줄 알았는데 String 이어서 2단계에서 곤욕을 치뤘다.
- 아무것도 몰랐다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.

- 웹은 재밌지만 시간이 오래 걸린다.
- 알렉스의 마법을 보고 반했다.

**Comment by asebn1** [comment]
# 바니와 제로
**1. 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?**
  - 모르는 내용이라 이상한 곳에서 헤맸다. 공부해야겠다.
  - 요구사항을 다시 한번 잘 읽어봐야겠다. 프론트엔드 어렵다.
  
**2. 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.**
  - 어렵다. 더 공부해야겠다.
  - 체스 미션이 두렵다.
  - HTML공부를 더 해야겠다.

**Comment by Ohzzi** [comment]
## 오찌 & 스컬
### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- (2단계까지 진행하면서) json을 쓸 줄 알았는데 플레인텍스트로 전달해줘서 그 부분에서 상당히 헤맸다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- api 설계를 정확히 알아보고 코드를 작성해야 한다는 것을 느꼈다.

**Comment by hyunrrr** [comment]
## 페퍼 & 조시

### 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?
- html 태그의 속성들이 햇갈렸다.
- html을 처음 해봐서 어떻게 접근해야하는지 어려웠습니다.

### 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.
- 프론트를 구현한다 해도 백엔드 코드를 잘 이해해야 될거 같다.
- 체스미션 너무 어려울거 같다. 벌써 걱정이다.

**Comment by yaho99** [comment]
# 아스피 & 야호

## 오늘 강의와 미션을 진행하면서 어디서 어려운 점을 느꼈나요?

- get 과 post 의 차이가 어려웠다.
- name 과 id 의 차이가 어려웠다.

## 오늘 진행하면서 느낀 점과 피드백을 자유롭게 작성해주세요.

- 다 했던 시도들인데 name만 안바꿔서 실패한게 너무 억울했다.
- id 만 command 로 하고 name 을 안바꿔서 억울했다.
- 그냥 너무 억울하다.
